infrastructures/infrastructure-factory/src/main/java/org/eclipse/che/api/factory/server/scm/kubernetes/KubernetesGitCredentialManager.java

```diff
@@ -54,18 +56,22 @@
   public static final String ANNOTATION_SCM_URL = "che.eclipse.org/scm-url";
   public static final String ANNOTATION_SCM_USERNAME = "che.eclipse.org/scm-username";
   public static final String ANNOTATION_CHE_USERID = "che.eclipse.org/che-userid";
-
+  public static final String CREDENTIALS_MOUNT_PATH = "/home/theia/.git-credentials";
+  public static final String LABEL_DEV_WORKSPACE_CREDENTIAL =
+      DEV_WORKSPACE_PREFIX + "/git-credential";
   private static final Map<String, String> LABELS =
       ImmutableMap.of(
-          "app.kubernetes.io/part-of", "che.eclipse.org",
-          "app.kubernetes.io/component", "workspace-secret");
+          "app.kubernetes.io/part-of",
```

Comment:
can we keep old formatting? It's a map, so having `key`, `value` on same line looks better to me.

Reply:
Sure. I thought maven fmt would put it back for me :)